css/properties/object-view-box.json

```
@@ -0,0 +1,39 @@
+{
+  "css": {
+    "properties": {
+      "object-view-box": {
+        "__compat": {
+          "spec_url": "https://drafts.csswg.org/css-images-4/#the-object-view-box",
```

Since `drafts.csswg.org` is down so often, we require the use of another official mirror instead:

```suggestion
          "spec_url": "https://w3c.github.io/csswg-drafts/css-images-4/#the-object-view-box",
```